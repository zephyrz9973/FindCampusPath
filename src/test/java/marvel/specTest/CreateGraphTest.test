#Create the graph
LoadGraph graph1 marvelTest.tsv

#list nodes of the graph
ListNodes graph1

#list child node of one node
ListChildren graph1 Cat

#list child node of another node
ListChildren graph1 Jellyfish

#list child node of last node
ListChildren graph1 Turtle