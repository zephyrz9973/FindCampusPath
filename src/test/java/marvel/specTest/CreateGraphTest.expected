#Create the graph
loaded graph graph1

#list nodes of the graph
graph1 contains: Cat Jellyfish Pig Snake Turtle

#list child node of one node
the children of Cat in graph1 are: Pig(Mammal) Pig(Vertebrate) Turtle(Vertebrate)

#list child node of another node
the children of Jellyfish in graph1 are:

#list child node of last node
the children of Turtle in graph1 are: Cat(Vertebrate) Pig(Vertebrate) Snake(Reptiles)